orchagent/portsorch.cpp

```diff
@@ -6618,13 +6618,13 @@ void PortsOrch::doTask(NotificationConsumer &consumer)
 
 void PortsOrch::updatePortOperStatus(Port &port, sai_port_oper_status_t status)
 {
-    SWSS_LOG_NOTICE("Port %s oper state set from %s to %s",
-            port.m_alias.c_str(), oper_status_strings.at(port.m_oper_status).c_str(),
-            oper_status_strings.at(status).c_str());
     if (status == port.m_oper_status)
     {
         return;
     }
+    SWSS_LOG_NOTICE("Port %s oper state set from %s to %s",
```

Comment:
lgtm, but why would this be notified to orchagent if there is no change in status?

Reply:
I am not sure I understand.
The current behavior was that it was being notified even when status is not being changed and followed with returning from the function. I suggested to have first the check if port status is actually being changed - in case it doesn't nothing will be notified as status wasn't changed.

Reply:
@arfeigin what about updating the DB unnecessarily even though there is no actual status change?

Reply:
You are correct, I think a fix for that will be that I will change the function to return a bool indicating if port status was actually changed and only if it was actually changed the DB will be updated.
Shall I add it to the PR?

Reply:
@arfeigin can you please check why there are OA notification from your platform or SDK even though the port operational status has not changed?